retriever/engines/mysql.py

```diff
@@ -39,6 +41,21 @@ class engine(Engine):
                       "{db}.{table}"),
                      ]
 
+    def check_credentials(self):
+        self.get_input()
+        self.sql_lookup = {"user": self.opts["user"], "password": self.opts["password"], "host": self.opts["host"], "port": self.opts["port"]}
```

Comment:
@jainamritanshu, This is looking good. However, I think we can remove the dictionary `sql_lookup`. What do you think.
From what I see, we are creating an extra dictionary sql_lookup but we can just update the  self.opts dictionary.
Let me know if I am missing something